app-misc/bgrep/bgrep-1_p20210321.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+GITHUB_USER="tmbinc"
+COMMIT="58c0b83f7ef13cd8d3d7352fadef88a006a514cd"
+
+inherit toolchain-funcs
+
+DESCRIPTION="Grep-like tool to search for binary strings"
+HOMEPAGE="https://github.com/tmbinc/bgrep/"
+SRC_URI="https://github.com/tmbinc/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
```

Comment:
Also please get in to habit of not typing `${PN}` here. In this case even it doesn't save ANY space from the string.
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D